src/core/renderer/hostDescriptors/descriptors/geometries/latheGeometry.ts

```diff
@@ -0,0 +1,50 @@
+import * as THREE from "three";
+import {LatheGeometry} from "three";
+import {GeometryContainerType, GeometryWrapperBase} from "../../common/geometryBase";
+import {IThreeElementPropsBase} from "../../common/IReactThreeRendererElement";
+import {WrappedEntityDescriptor} from "../../common/ObjectWrapper";
+
+export interface ILatheGeometryProps {
+  points: Array<THREE.Vector2>;
+  segments?: number;
+  phiStart?: number;
+  phiLength?: number;
+}
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      latheGeometry: IThreeElementPropsBase<THREE.LatheGeometry> & ILatheGeometryProps;
+    }
+  }
+}
+
+export class LatheGeometryWrapper extends GeometryWrapperBase<ILatheGeometryProps, LatheGeometry> {
+  protected constructGeometry(props: ILatheGeometryProps): LatheGeometry {
+    const points = props.points.map(p => new THREE.Vector3(p.x, p.y, 0.0))
+    return new LatheGeometry(
+      points,
```

Comment:
Hmm... looking at lathegeometry.js I see that only .x and .y properties are used. So it should be safe to pass in `props.points as any` and add a comment on why that's being done :)

And then you can make a PR in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/three/three-core.d.ts#L6647 to change from `Vector3[]` to `Vector2[]` :D

Reply:
See also: https://github.com/mrdoob/three.js/blob/dev/src/geometries/LatheGeometry.js#L89-L91

Reply:
Sounds good, expect inc changes soon :)